doc/releases/release-notes-4.0.rst

```diff
@@ -299,6 +299,9 @@ Libraries / Subsystems
 
 * Crypto
 
+  * Mbed TLS was updated to version 3.6.1. The release notes can be found at:
+    https://github.com/Mbed-TLS/mbedtls/releases/tag/mbedtls-3.6.1
```

Comment:
this will fail on 3.7 backport

Reply:
What will happen? Should something be done about it now?

Reply:
Is the goal to backport this to 3.7?  Asking as I don't see that mentioned in the PR.  Either way, it would be good to have Zephyr LTS continue to track the mbedTLS LTS version.

Reply:
yep, LTS should keep MBEDTLS updated.

Reply:
btw, I wouldn't add anything in the release notes right now since we may end up updating it before our release in cases there is another MBEDTLS release.

Reply:
> Is the goal to backport this to 3.7? Asking as I don't see that mentioned in the PR. Either way, it would be good to have Zephyr LTS continue to track the mbedTLS LTS version.

Absolutely. I have added the `backport v3.7-branch` label. From that you can see that a backport is planned.

Reply:
> btw, I wouldn't add anything in the release notes right now since we may end up updating it before our release in cases there is another MBEDTLS release.

Yeah true in a sense, that would also have prevented conflicts with backporting as @nordicjm pointed out. But now it's done and there are approvals so let's go with this. The release notes can simply be updated if/when Mbed TLS is updated again. At least the release note is already there.